crypto/fipsmodule/ec/oct.c

```diff
@@ -110,6 +121,14 @@ size_t ec_point_to_bytes(const EC_GROUP *group, const EC_AFFINE *point,
     uint8_t y_buf[EC_MAX_BYTES];
     ec_felem_to_bytes(group, y_buf, &field_len, &point->Y);
     buf[0] = form + (y_buf[field_len - 1] & 1);
+    if (form == POINT_CONVERSION_HYBRID) {
+      // |POINT_CONVERSION_HYBRID| specifies y's solution of the quadratic
+      // equation, but also encodes the y coordinate along with it.
+      OPENSSL_memcpy(buf + 1 + field_len, y_buf, field_len);
+      if (!is_hybrid_bytes_consistent(buf, field_len)) {
```

Comment:
there is no need for this check here because it can't fail. It can't fail because of how you set `buf[0]` in line 123: `buf[0] = form + (y_buf[field_len - 1] & 1);`